libevmasm/PeepholeOptimiser.cpp

```diff
@@ -117,6 +117,35 @@ struct OpPop: SimplePeepholeOptimizerMethod<OpPop>
 	}
 };
 
+struct OpStop: SimplePeepholeOptimizerMethod<OpStop>
```

Comment:
Is this not handled by other steps?

Reply:
Apparently not...
Ideally, this wouldn't be needed, but apparently some cases still slip through the logic below... and it generally doesn't hurt to have this.
It fixes the regression on the ``FixedRegistrar`` semantics test and saves ~2% of code size there apparently... this warrants a closer look at some point, but for now the peephole optimizer seems like a good fallback mechanism.